conans/client/graph/proxy.py

```diff
@@ -113,6 +113,11 @@ def _download_recipe(self, layout, ref, output, remotes, remote, recorder):
         def _retrieve_from_remote(the_remote):
             output.info("Trying with '%s'..." % the_remote.name)
             # If incomplete, resolve the latest in server
+            if "https://conan.bintray.com" in the_remote.url:
```

Comment:
I think we should declare that URL as a global variable in this module like:

```python
BINTRAY_DEPRECATED_URL = "https://conan.bintray.com"

# .....

if BINTRAY_DEPRECATED_URL in the_remote.url:
      # whatever
```
Then, it should be easier to use it in any test.